plugins/tomcat_logs.yaml

```diff
@@ -0,0 +1,90 @@
+version: 0.0.1
+title: Apache Tomcat
+description: Log parser for Apache Tomcat
+parameters:
+  - name: enable_access_log
+    type: bool
+    default: true
+  - name: access_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/tomcat/logs/localhost_access_log.*.txt"
+  - name: enable_catalina_log
+    type: bool
+    default: true
+  - name: catalina_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/tomcat/logs/catalina.out"
+  - name: start_at
+    type: string
+    supported:
+     - beginning
+     - end
+    default: end
+template: |
+  receivers:
+    {{ if .enable_access_log }}
+    filelog/access:
+      include:
+        {{ range $fp := .access_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: tomcat.access
+      operators:
+        - type: regex_parser
+          regex: '(?P<remote_host>[^\s]+) - (?P<remote_user>[^\s]+) \[(?P<timestamp>[^\]]+)\] "(?P<method>[A-Z]+) (?P<path>[^\s]+)[^"]+" (?P<status>\d+) (?P<bytes_sent>[^\s]+)'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%d/%b/%Y:%H:%M:%S %z'
+          severity:
+            parse_from: attributes.status
+            preserve_to: attributes.status
```

Comment:
I don't think this field is needed. It's not listed in the [severity operator docs](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/types/severity.md). Probably worth testing without it just to verify.

Reply:
Yes, `preserve_to` was removed, and the default behavior is that the field is preserved to itself, so this field can be removed.